src/ripple/app/tx/impl/PayChan.cpp

```diff
@@ -496,20 +714,20 @@ PayChanClaim::doApply()
         }
 
         if (reqBalance > chanFunds)
-            return tecUNFUNDED_PAYMENT;
+            return tecINSUFFICIENT_FUNDS;
```

Comment:
should this be guarded by amendment flag or no?
```
if (ctx_.view().rules().enabled(featurePaychanAndEscrowForTokens))
    return tecINSUFFICIENT_FUNDS;
```

Reply:
Looks like it yes. We will make this change.

Reply:
This should actually be `tecUNFUNDED_PAYMENT` and it should match the tests that existed before the amendment. This was my error.